### DISCUSSION TITLE
[M-Z] (2024/25) esercizi per casa 8 ottobre (anelli)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#490] [Esercizi M-Z]
![es.08-10 ](https://github.com/user-attachments/assets/7b42d8dd-6029-4047-a63b-6b99a9e4a368)
![08-10,2](https://github.com/user-attachments/assets/63e76beb-ca5c-4241-8a64-871689ee82cf)

**Comment by AglaiaNorza** [comment]
io li ho fatti così (ma non assicuro niente)
(tranne il 4 - avevo svolto la seconda parte in maniera errata)
![1-2](https://github.com/user-attachments/assets/d889fb14-ffb9-4192-a108-89a970954ede)
![3](https://github.com/user-attachments/assets/7464e2e3-37ae-4171-bb96-95f3e7ca6da0)

![image](https://github.com/user-attachments/assets/240018d5-bf28-44db-8882-73a265fe111f)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/414d6970-9ea6-40e6-813c-d537ea439e4f)
![immagine](https://github.com/user-attachments/assets/7bcb977e-9815-4f7a-9907-1cbf8035ed0f)
![immagine](https://github.com/user-attachments/assets/0c5a5be3-7794-4d48-ab85-35d2b6d3022a)
